common/repositories.sh

```diff
@@ -32,6 +32,7 @@ sawtooth_repositories=(
   https://github.com/hyperledger/sawtooth-next-directory.git
   https://github.com/hyperledger/sawtooth-pbft.git
   https://github.com/hyperledger/sawtooth-poet.git
+  https://github.com/hyperledger/sawtooth-private-utxo.git
```

Comment:
This says ARCHIVED. Thus I removed it.

Reply:
I transferred it to the archive and updated this line